app/services/trainees/create_from_hesa.rb

```diff
@@ -167,7 +168,7 @@ def disability
     end
 
     def enqueue_background_jobs!
-      if FeatureService.enabled?(:integrate_with_dqt) && trainee.trn.blank?
+      if integrate_with_dqt && trainee.trn.blank?
```

Comment:
I think we should always do this as we need to set the state. The RegisterForTrnJob can worry about whether its enabled or not.